core/storage/suggestion/gae_models.py

```diff
@@ -693,6 +693,15 @@ def get_in_review_translation_suggestions_with_exp_ids_by_offset(
             language_codes: list(str). The list of language codes.
             exp_ids: list(str). Exploration IDs matching the target ID of the
                 translation suggestions.
+            Maximum number of entities to be returned: Note that the None case
```

Comment:
This is still in the wrong place. Please don't invent new docstring formats. The place you're putting it makes it look like a new argument.

Put this explanation in the docstring for the specific argument that it pertains to. If it pertains to the return value then put it there instead.

Reply:
Okay, I got confused regarding it's placement, fixed it now. Thanks.